apps/discord/commands.py

```
@@ -0,0 +1,108 @@
+"""Discord slash command definitions and handlers."""
+
+from typing import Dict, List
+
+import aiohttp
+from config import ALLOWED_GUILD_IDS, DiscordResponseType
```

Ruff (0.12.2)

`config.DiscordResponseType` imported but unused

Remove unused import: `config.DiscordResponseType`

(F401)